gdctx/store.go

```diff
@@ -15,14 +19,25 @@ func RegisterStorePrefix(prefix string) {
 }
 
 // InitStore is to initialize the store
-func initStore() {
-	Store = store.New(Restart)
+func InitStore(initPrefix bool) {
+	Store = store.New()
+
+	if initPrefix == true {
```

Comment:
The linter should have complained about this. You don't need to check for `boolval == true` for an `if` statement. Keep it `if initPrefix {`.